contributors/devel/on-call-federation-build-cop.md

```diff
@@ -0,0 +1,159 @@
+# Federation Buildcop Guide and Playbook
+
+Federation runs two classes of tests: CI and Presubmits.
+
+## CI
+
+* These tests run on the HEADs of master and release branches.
+* As a result, they run on code that's already merged.
+* As the name suggests, they run continuously. Currently, they are
+  configured to run at least once every 30 minutes.
+* Federation CI tests still run on Jenkins and this mode of testing is
+  now deprecated.
+* CI jobs always run sequentially. In other words, no single CI job
+  can have two instances of the job running at the same time.
+
+### Configuration
+
+Configuration steps are described in https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jenkins/README.md#how-to-work-with-jenkins-jobs
+
+The configuration of CI tests are stored in:
+
+* Jenkins config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml
+* Test job/bootstrap config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jobs/config.json
+* Test grid config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/testgrid/config/config.yaml
+* Job specific config: https://github.com/kubernetes/test-infra/tree/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jobs
+
+### Results
+
+Results of all the federation CI tests, including the soak tests, are
+listed in the corresponding tabs on the Cluster Federation page in the
+testgrid.
+https://k8s-testgrid.appspot.com/cluster-federation
+
+### Playbook
+
+#### Triggering a new run
+
+Please ping someone who has access to the Jenkins UI/dashboard and ask
+them to login and click the "Build Now" link on the Jenkins page
+corresponding to the CI job you want to manually start.
+
+#### Quota cleanup
+
+Please ping someone who has access to the GCP project and ask them to
```

Comment:
How do I find someone who has access? And what does someone who has access do? Is there a script they need to run?